huxley/api/permissions.py

```diff
@@ -222,6 +223,40 @@ def has_permission(self, request, view):
         return user_is_advisor(request, view, school_id)
 
 
+class CommitteeFeedbackListPermission(permissions.BasePermission):
+    '''Accept GET for only the chair of the committee'''
+
+    def has_permission(self, request, view):
+        user = request.user
+        if user.is_superuser:
+            return True
+
+        method = request.method
+        committee_id = request.query_params.get('committee', -1)
+        return method == 'GET' and user_is_chair(request, view, committee_id)
+
+
+class CommitteeFeedbackDetailPermission(permissions.BasePermission):
+    '''Accept POST for only the delegate of the committee'''
+
+    def has_permission(self, request, view):
+        user = request.user
+        if user.is_superuser:
+            return True
+
+        method = request.method
+        committee_id = view.kwargs.get('committee', -1)
+        if method == 'POST' and user.is_authenticated() and user.is_delegate(
+        ) and user.delegate.assignment:
```

Comment:
Breaking the parentheses here feels awkward; perhaps try splitting at each 'and' instead

Reply:
Lol yea I agree, I think the autoformatter did that one